furnace.c

```diff
@@ -49,9 +50,10 @@ typedef struct {
    *
    * TODO: Handle pwm_deadline overflow. Should we handle it?
    */
-  uint64_t pwm_level    : 4;
+  struct pid_ctx *pid;
```

Comment:
Can we place it somewhere else in the structure? The above comment is for all of the `pwm_*` fields and now it looks like it also apply to `pid`.